makeservices/makemysql-real

```diff
@@ -155,85 +80,105 @@ def main():
             raise RuntimeError('Unable to read SUDO_USER.')
 
         # Read config file.
-        read_config()
+        mysql_host, mysql_root_pw = read_config()
 
         # Check whether the script should proceed.
-        intro_prompt()
+        if not intro_prompt():
+            print('>>> Aborted by user request.')
+            return
 
         # Connect to the MySQL server.
         try:
             print('>>> Connecting to MySQL database server...')
-            connection = MySQLdb.connect(host=MYSQL_HOST,
+            connection = MySQLdb.connect(host=mysql_host,
                                          user='root',
-                                         passwd=MYSQL_ROOT_PW)
-        except:
+                                         passwd=mysql_root_pw)
+        except Exception as ex:
             print('>>> Error: Failed to connect to MySQL server.')
-            sys.exit(1)
+            raise ex
```

Comment:
`raise` by itself will re-raise the same exception, so it will still work.

you can actually just write in this case:

```python:
except:
    print('>>> Error: Failed to connect to MySQL server.')
    raise
```